l10n_us_check_writing_address/report/print_check.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Ursa Information Systems <http://www.ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo.osv import osv
+from odoo.report import report_sxw
+from odoo import api, fields, models, _
+
+LINE_FILLER = '*'
+INV_LINES_PER_STUB = 9
+
+
+class ReportPrintCheck(report_sxw.rml_parse):
+
+    def __init__(self, cr, uid, name, context):
+        super(report_print_check, self).__init__(cr, uid, name, context)
+        self.localcontext.update({
+            'pages': self.get_pages,
+        })
+
+    def fill_line(self, amount_str):
+        return amount_str and (amount_str+' ').ljust(200, LINE_FILLER) or ''
+
+    def get_pages(self, payment):
+        """ Returns the data structure used by the template : a list of dicts containing what to print on pages.
+        """
+        stub_pages = self.make_stub_pages(payment)
+        multi_stub = payment.company_id.us_check_multi_stub
+        pages = []
+        for i in range(0, stub_pages != None and len(stub_pages) or 1):
+            pages.append({
+                'sequence_number': payment.check_number\
```

Comment:
This conditional is too complex to be inside of a dictionary assignment. Please break out into a temporary variable

My general rule of thumb is that `\` is a dirty hack, and any time it's used is a sign of a refactor need.